staging/src/k8s.io/legacy-cloud-providers/azure/azure_storageaccount_test.go

```diff
@@ -84,3 +84,171 @@ func TestGetStorageAccessKeys(t *testing.T) {
 		}
 	}
 }
+
+func TestGetStorageAccount(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	cloud := &Cloud{}
+
+	name := "testAccount"
+	location := "testLocation"
+	networkID := "networkID"
+	accountProperties := storage.AccountProperties{
+		NetworkRuleSet: &storage.NetworkRuleSet{
+			VirtualNetworkRules: &[]storage.VirtualNetworkRule{
+				{VirtualNetworkResourceID: &networkID,
```

Comment:
nit: please leave `{` and `}` in one separated line

Reply:
done